services/management/ia/templates_service.go

```diff
@@ -464,14 +464,15 @@ func (s *TemplatesService) UpdateTemplate(ctx context.Context, req *iav1beta1.Up
 		return nil, status.Error(codes.InvalidArgument, "Request should contain exactly one rule template.")
 	}
 
-	for _, t := range templates {
-		if err = validateUserTemplate(&t); err != nil { //nolint:gosec
-			return nil, status.Errorf(codes.InvalidArgument, "%s.", err)
-		}
+	tmpl := templates[0]
+
+	if err = validateUserTemplate(&tmpl); err != nil { //nolint:gosec
+		return nil, status.Errorf(codes.InvalidArgument, "%s.", err)
 	}
 
 	params := &models.ChangeTemplateParams{
```

Comment:
If there are more params variables in the function, it would be good to distinguish them by adding an info to the name.

Reply:
What do you mean? There is just one param in function and that is struct. If you mean commen.
I don't see point to add comment to self-explanatory fields. You can't comment every field of entity.

Reply:
Line 452 there is pParams. Now here just params. They could have more info in the name.
Funny that adding a few letters to the name is 1 - 2 secs of typing. But how much time do these discussions on PRs take? Is it really so beneficial to save those two seconds when naming the variable?

Reply:
Sorry, you're right. I'd gotten out of the designated green area.